<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>515</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>:)</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:  #B7CEE8;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tableView">
      <property name="styleSheet">
       <string notr="true">background-color:  white;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelTabel">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Таблица</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbTable">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	text-align: center; 
    border-radius: 15px;
    min-width: 6em;
	background: white;
}

QComboBox:editable {
    background: white;
}


QComboBox:on { /* shift the text when the popup opens */
    padding-top: 50px;
    padding-left: 50px;
	    background: white;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}


QComboBox QAbstractItemView {
    background: white; 
	border-radius: 10px;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelView">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Представления</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbView">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	text-align: center; 
    border-radius: 15px;
    min-width: 6em;
	background: white;
}

QComboBox:editable {
    background: white;
}


QComboBox:on { /* shift the text when the popup opens */
    padding-top: 50px;
    padding-left: 50px;
	    background: white;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}


QComboBox QAbstractItemView {
    background: white; 
	border-radius: 10px;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="labelFontSize">
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Размер шрифта</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="cmbFontSize">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QComboBox {
	text-align: center; 
    border-radius: 15px;
    min-width: 6em;
	background: white;
}

QComboBox:editable {
    background: white;
}


QComboBox:on { /* shift the text when the popup opens */
    padding-top: 50px;
    padding-left: 50px;
	    background: white;
}

QComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 15px;

    border-left-width: 1px;
    border-left-color: darkgray;
    border-left-style: solid; /* just a single line */
    border-top-right-radius: 3px; /* same radius as the QComboBox */
    border-bottom-right-radius: 3px;
}

QComboBox::down-arrow:on { /* shift the arrow when popup is open */
    top: 1px;
    left: 1px;
}


QComboBox QAbstractItemView {
    background: white; 
	border-radius: 10px;
}</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnDelete">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>33</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: white; border-radius: 15px;</string>
      </property>
      <property name="text">
       <string>Удаление записи</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionLoginShow"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoginShow">
   <property name="text">
    <string>Подключение к БД</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
